patroni/__main__.py

```diff
@@ -14,6 +14,7 @@
 
 from patroni import MIN_PSYCOPG2, MIN_PSYCOPG3, parse_version
 from patroni.daemon import abstract_main, AbstractPatroniDaemon, get_base_arg_parser
+from patroni.dcs import Cluster
```

Comment:
Please move it under `if TYPE_CHECKING:`:
```python
if TYPE_CHECKING:  # pragma: no cover
    from .config import Config
    from .dcs import Cluster
```

After that `Cluster` type should be referenced as string literal: `'Cluster'`.